### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by Khalniepiesova**
- [DEMO LINK](Khalniepiesova.github.io/layout_catalog_grid/)

### CODE REVIEW

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
Hi, fix your DEMO link and request review again
image

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Frontend practice with catalog page using grid
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog_grid/)
+- [DEMO LINK](Khalniepiesova.github.io/layout_catalog_grid/)
```
[the remark this answers is not in the document]
Comment (DarkMistyRoom) [reply readme.md:3]: fix the demo link in the description. it still doesn't work

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,14 +1,43 @@
...
+  .catalog__cards {
+    grid-template-columns: 200px 200px;
+  }
+}
+
+@media (max-width: 487px) {
+  .catalog__cards {
+    grid-template-columns: 200px;
+  }
+}
```
Comment (DarkMistyRoom) [inline src/styles/main.scss:43]: looks like a BEM block. so move it to a separate file

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,14 +1,43 @@
...
 
 body {
   margin: 0;
 }
 
-* {
-  box-sizing: border-box;
+.catalog__cards {
+  display: grid;
+  grid-template-columns: 200px 200px 200px 200px;
```
Comment (DarkMistyRoom) [inline src/styles/main.scss:17]: use `repeat`

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Good job. Remember about file structure

File: src/styles/main.scss (hunk 2 above)
Comment (Khalniepiesova) [reply src/styles/main.scss:43]: When I moved @media to a separate file  it does not work

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,14 +1,43 @@
...
+  .catalog__cards {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (max-width: 487px) {
+  .catalog__cards {
+    grid-template-columns: 200px;
+  }
+}
```
Comment (DarkMistyRoom) [inline src/styles/main.scss:43]: not fixed

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

**Review by ViktorHavryliuk — CHANGES_REQUESTED** [review]
Demo link not fixed
image

File: src/styles/blocks/header.scss [hunk 5]
```diff
@@ -0,0 +1,63 @@
+@import "./utils/variables.scss";
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: $color-light;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
```
Comment (DarkMistyRoom) [inline src/styles/blocks/header.scss:9]: do you remember the requirement from the moyo-header task about height? don't set it directly to the header

File: src/styles/blocks/header.scss [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  display: flex;
+  height: 60px;
+  align-items: center;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  margin: 0;
+  padding: 0;
+  cursor: pointer;
+}
```
Comment (DarkMistyRoom) [inline src/styles/blocks/header.scss:46]: create the nav selector and use nesting

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

**Pull request closed without merging**